modules/nf-core/proteinortho/main.nf

```diff
@@ -0,0 +1,54 @@
+process PROTEINORTHO {
+    tag "$meta.id"
+    label 'process_high'
```

Comment:
Does this need process high?

Reply:
Proteinortho scales quadratically with input fasta files and each iteration is using e.g. DIAMOND which is already in process_medium. 
process_high uses 12 cores, which I strongly recommend otherwise this can take a long long time to complete medium-sized requests, like the QfO benchmark dataset.

Reply:
Ok, that's fine, if it generally needs high and scales well then assign it high.